doc/ext/traffic-server.py

```diff
@@ -188,7 +188,7 @@ class TSStat(std.Target):
 
     option_spec = {
         'type': metrictypes,
-        'unit': metricunits,
+        'units': metricunits,
```

Comment:
Shouldn't they both be plural or singular (type and unit)?

Reply:
They get converted to something else for display. For instance, in the statistics pages that amc has changed, we show "Datatype:". (https://docs.trafficserver.apache.org/en/latest/admin-guide/monitoring/statistics/core/general.en.html)

Reply:
If that's ok by you bcall, I'd approve this.

Reply:
I can change that too if you like, but the "unit" / "units" difference causes actual documentation building problems as a trap for unwary contributors. I haven't had to clean up anything involving the "type" option.